### PR TITLE
Refactored unit tests

### PR DESCRIPTION
**Pull request by AlexandrouR**
Refactored the tests to improve the time.
Fixed generate-tests to show defaults

### CODE REVIEW

**Comment by cfelde** [comment]
There's a few non-test class changes in here as well?

File: src/main/resources/Template.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ private static Web3j createWeb3jService(String url){
             if (System.getProperty("NODE_URL") == null || System.getProperty("NODE_URL").isEmpty()) {
                 if (System.getenv("NODE_URL") == null || System.getenv("NODE_URL").isEmpty()) {
                     log.info("Please make sure the node url is valid.");
-                    log.info("You can edit the node url programmatically, use java -DnodeURL=\"\" or as an environmental variable e.g export NODE_URL=\"\"");
+                    log.info("You can edit the node url programmatically, use java -DNODE_URL=\"\" or as an environmental variable e.g export NODE_URL=\"\"");
```
Comment (cfelde) [inline src/main/resources/Template.java]: Again I think we should have the NODE_URL value defined in a static final String. Does this change imply that v4.5.11 is showing the wrong -D output?

Comment (AlexandrouR) [reply src/main/resources/Template.java]: On it.
Yes it shows the wrong output

**Comment by cfelde** [comment]
LGTM

**Pull request merged**